cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -48,24 +49,32 @@ func loadConfig(data []byte) (*kubeschedulerconfig.KubeSchedulerConfiguration, e
 	return nil, fmt.Errorf("couldn't decode as KubeSchedulerConfiguration, got %s: ", gvk)
 }
 
-// WriteConfigFile writes the config into the given file name as YAML.
-func WriteConfigFile(fileName string, cfg *kubeschedulerconfig.KubeSchedulerConfiguration) error {
+// LogAndWriteConfigFile writes the config into the given file name as YAML.
+func LogAndWriteConfigFile(fileName string, cfg *kubeschedulerconfig.KubeSchedulerConfiguration) error {
```

Comment:
Not sure who would use a filename (probably just us k8s-devs), but why not "LogOrWrite"